system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -169,8 +169,9 @@
     // We then transition on max-height, since we can't transition height to/from auto.
     max-height: 900px;
 
-    // This is so the top sites favicon doesn't get clipped during animation:
-    margin-inline-end: -7px;
+    // This is so the top sites favicon and card dropshadows don't get clipped during animation:
+    margin: 0 -7px;
+    padding: 0 7px;
```

Comment:
nit: define a variable right here to use `margin: 0 -$var` and `padding: 0 $var` to be clear they're opposites.